Showcase_Apps/README.md

```diff
@@ -7,6 +7,7 @@ All of the applications in these folders are full working applications.  Please
 
 ## Applications
 * Image Handling
+* FormCell Inline Validation
```

Comment:
Can you move "FormCell Inline Validation" before "Image Handling" to have those showcase apps (no extensions) in an alphabetical order?

Reply:
Done